state/cloudcredentials.go

```diff
@@ -386,7 +386,7 @@ func validateCloudCredentials(
 }
 
 func validateCredentialForCloud(nuage cloud.Cloud, tag names.CloudCredentialTag, credential Credential) error {
-	if tag.Cloud().Id() != nuage.Name {
+	if tag.Cloud().Id() != nuage.Name && nuage.Name != "manual" {
```

Comment:
Don't we want both the source cloud and target cloud to be "manual"?

Reply:
you're right, this change was a left-over and should have been removed.